portfolio/src/main/webapp/index.html

```diff
@@ -166,9 +166,17 @@ <h3>
       <div>
         <h1>Contact Me</h1>
         <br />
-        <p>Send me an email at zoe.barth@gmail.com!</p>
-        <button onclick="fetchContent()">Fetch content</button>
-        <div id="content-container"></div>
+        <p>
+          Feel free to write me an email at zoe.barth@gmail.com or leave a
+          comment below:
+        </p>
+        <form action="/data" method="POST">
+          <textarea id="comment" name="comment"> </textarea>
+          <br />
+          <br />
```

Comment:
boo 👎  you can do better
use margin/padding for spacing, not `<br />`

Reply:
I changed it to only one <br /> and added margin to the bottom. I'm open to getting rid of that one <br />, but I don't know how to do it without completely messing up the layout of the form.

Reply:
ok, sounds like a separate issue we can address if need be
sounds good to me